### DISCUSSION TITLE
Is 'upload' a reserved keyword for component properties?

### DISCUSSION THREAD

**Discussion by rahulmkhj** [livewire/livewire#8954] [1. Help] [answered]
Hi. While building a livewire component I did `wire:mode="upload"` and suddenly the value on the text input is set to: `(...params) => upload(component, ...params)` changing the property name from upload to anything else fixes this. 

Wonder if there is a list of reserved keywords in livewire that we are not supposed to use? And if `upload` is among those keywords?

Otherwise, I will report this as a bug.

**Comment by joshhanley** [comment, marked as answer]
@rahulmkhj yep it's reserved. See the warning on the file upload page, straight under the first code example 🙂 https://livewire.laravel.com/docs/uploads

<img width="733" alt="image" src="https://github.com/user-attachments/assets/e24b6bcb-6239-4dc3-abc1-b8330facb823">
